bin/fin

```diff
@@ -850,9 +850,9 @@ _vhost_proxy_connect ()
 	docker network connect "$network" docksal-vhost-proxy >/dev/null 2>&1
 	if [[ $? == 0 ]]; then
 		echo-green "Connected vhost-proxy to \"${network}\" network."
+		# Run a dummy container to trigger docker-gen to refresh proxy configuration.
```

Comment:
Propose: 
# Run a dummy container to trigger docker-gen inside vhost-proxy to generate proxy configuration.